### PR TITLE
feat(pack): add Harper 

### PR DESCRIPTION
**Pull request by cristobalgvera**
## 📑 Description

Add [`harper-ls`](https://github.com/elijah-potter/harper/tree/master/harper-ls) integration, a `LSP` to interop with [Harper](https://writewithharper.com).

## ℹ Additional Information

From it's website:

> Harper is an English grammar checker designed to be just right. You can think of it as an open-source alternative to Grammarly.

### CODE REVIEW

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks for the contribution! This should be a language pack that goes in `lua/astrocommunity/pack/harper.lua` since it configures multiple plugins at once time and also is not providing a new plugin.

**Review by mehalter — COMMENTED** [review]
This block should be moved up to a top level spec rather than a dependency. If the user has disabled Mason, the spec should still apply the recommended settings to the language server

File: lua/astrocommunity/utility/harper-ls/init.lua [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                  linters = {
+                    spelled_numbers = true,
+                    linking_verbs = true,
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
```
[the remark this answers is not in the document]
Comment (cristobalgvera) [reply lua/astrocommunity/utility/harper-ls/init.lua]: Thanks for this comment, I didn't know about it!

**Comment by cristobalgvera** [comment]
> Thanks for the contribution! This should be a language pack that goes in `lua/astrocommunity/pack/harper.lua` since it configures multiple plugins at once time and also is not providing a new plugin.

Moved!

**Review by mehalter — CHANGES_REQUESTED** [review]
Thanks so much! Only thing is it should be `pack/harper` and not `pack/harper-ls`

**Review by mehalter — APPROVED** [review]
I went ahead and fixed it to get this merged. Thanks again!

**Pull request merged**